mattermost/mattermost-server
# Ensure feedback options are not submitting in their translated forms.

#### Summary

In the community notifications, we see the Delete Reason value as the translation option value selected in the delete workspace feedback.

The same might occur for the downgrade feedback.

Example of delete feedback: https://community.mattermost.com/private-core/pl/znxjthu69byeudydagomw1ysqe

#### Ticket Link

[MM-51692](https://mattermost.atlassian.net/browse/MM-51692)

#### Test Plan

**Ensure you have a license for a workspace with the following conditions:**
- Is subscribed to a cloud product
- Is on a free trial OR
- Is on a starter subscription OR
- Is on a monthly professional subscription

**For clarity, these subscriptions may be deleted:**
- Cloud-Starter.
- Cloud-Professional (monthly).
- Enterprise Free Trial.

**Steps For Testing**

1. Log into your workspace
2. Update your display settings and set `Language` to any language except for **`English`**.
3. When the language change has taken effect, navigate to `System Console > Billing & Account > Subscription`
4. Scroll to the bottom of the page and click the `Delete Workspace` button.
5. Go through the modal, deleting the workspace
6. Select any feedback option and submit
7. Ensure the notification in the Cloud Trial Lifecycle contains English text for the `Delete Reason` value.
  - **NOTE** Comments cannot be translated as they are defined by the user. 

#### Release Note

```release-note
NONE
```


[MM-51692]: https://mattermost.atlassian.net/browse/MM-51692?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
